lib/manageiq/appliance_console/postgres_admin.rb

```diff
@@ -62,13 +62,18 @@ def self.service_running?
       LinuxAdmin::Service.new(service_name).running?
     end
 
-    def self.local_server_in_recovery?
-      data_directory.join("recovery.conf").exist?
+    def self.local_server_received_standby_signal?
+      # Beginning with PostgreSQL 12, replication configuration has been integrated into the main PostgreSQL configuraton system and the conventional recovery.conf file is no longer valid.
+      # see: https://repmgr.org/docs/current/release-5.0.html
+      # https://www.2ndquadrant.com/en/blog/replication-configuration-changes-in-postgresql-12/
+      # “standby.signal” – indicates the server should start up as a hot standby
```

Comment:
Prefer "normal" double quotes so we don't have special characters in the source.

```suggestion
      # "standby.signal" – indicates the server should start up as a hot standby
```